stack/context.go

```diff
@@ -321,7 +322,7 @@ var (
 	reRaceGoroutine = regexp.MustCompile(`^Goroutine (\d+) \((running|finished)\) created at:$`)
 
 	// TODO(maruel): Use it.
-	//reRacePreviousOperationMainHeader = regexp.MustCompile("^Previous (read|write) at (0x[0-9a-f]+) by main goroutine:$")
+	// reRacePreviousOperationMainHeader = regexp.MustCompile("^Previous (read|write) at (0x[0-9a-f]+) by main goroutine:$")
```

Comment:
Is it your IDE that did that? I use this form for explicitly commented out code and would prefer to not change these lines here.

Reply:
Yeah sorry I can undo that. It defaults to adding spaces at the beginning of comments.